# S1
device_id:        0
fps:              15 # camera frame rate
image_width:      640 # camera image width
image_height:     480 # camera image height
dist_coeffs:      [-0.29363353, 0.0295368, -0.00107568, -0.00475553, 0.06122138]
camera_matrix:    [[383.82685631,  0.,           354.79497148],
                  [0.,            511.94164625, 160.47281803],
                  [0.,            0.,           1.          ]] # camera projection matrix

# S2
engine:           "/home/jetbot/EVC/final_project/src/final/src/2_identification/models/osnet_x0_5_fp16.engine"
ref_dir:          "/home/jetbot/EVC/final_project/src/final/src/2_identification/ref_person"
save_dir:         "/home/jetbot/EVC/final_project/src/final/src/2_identification/received_snipets"
save_enabled:     False
sim_threshold:    0.50
pub_rate:         0.2
hold_time:        0.2   # seconds to keep last ROI

# S3
kp_angular:         1.3
kd_angular:         0.7
max_ang:            3.0
deadband_pct:       0.01
forward_speed:      0.85
min_forward_pct:    0.3
forward_window_pct: 0.05
rate:               5 # dont increase above 5
smooth_alpha:       0.45
k1_speedup:         1.3
k2_slowdown:        0.35
backward_speed:    -0.5
history_size:       10
non_zero_history_size: 20
min_relative_img:   0.75
max_relative_img:   0.9
error:              0.9
smoothing_alpha:    0.5

# S4
frame_id:       "tof_link"
rate_obj:       10        # Hz
min_range:      0.03      # meters
max_range:      2.0       # meters
field_of_view:  0.015     # radians
threshold:      0.5      # meters
forward_speed:  0.35       # m/s
turn_speed:     4.5       # rad/s
window_size:      5       # number of measurements to average
detection_ratio:  1.0
turnleft_target:   4.5
forward_target:    20
turn_right_target: 4.5

# S5
rate_search:            1     # Hz
forward_speed_search:   0.5   # m/s
angular_speed_search:   0   # rad/s
state_manager_rate:     10